packages/eas-cli/src/credentials/ios/actions/new/SetupInternalProvisioningProfile.ts

```diff
@@ -1,12 +1,108 @@
-import { IosAppBuildCredentialsFragment } from '../../../../graphql/generated';
+import { IosAppBuildCredentialsFragment, IosDistributionType } from '../../../../graphql/generated';
+import Log from '../../../../log';
+import { promptAsync } from '../../../../prompts';
 import { Context } from '../../../context';
 import { AppLookupParams } from '../../api/GraphqlClient';
+import { getAllBuildCredentialsAsync } from './BuildCredentialsUtils';
 import { SetupAdhocProvisioningProfile } from './SetupAdhocProvisioningProfile';
+import { SetupProvisioningProfile } from './SetupProvisioningProfile';
 
 export class SetupInternalProvisioningProfile {
```

Comment:
Some description would be usefull here, both for what behaviour we implement, but also in what context it will be executed e.g, we assume that "enterpriseProvisioning" is not specified